mount/lookup_unix.go

```diff
@@ -30,17 +24,10 @@ func Lookup(dir string) (Info, error) {
 		return mounts[j].Mountpoint < mounts[i].Mountpoint
 	})
 	for _, m := range mounts {
-		// Note that m.{Major, Minor} are generally unreliable for our purpose here
```

Comment:
Please keep this comment, and also add a comment about not comparing Dev

Reply:
Sure. Will do.

Reply:
Done